### DISCUSSION TITLE
Feature request: Option for automatically splitting converted books

### DISCUSSION THREAD

**Discussion by ensoniq2k** [openaudible/openaudible#645] [Ideas]
Since I split all my converted books but have to do it manually I would love to see an automatice option. Ideally there would be some kind of marker for splitted books so they can be distinguished.

**Comment by openaudible** [comment]
That's an idea. But we're not a fan of splitting books. 
For now, you can now select all or some books and split them. OA should automatically skip books you've already split.

**Comment by ensoniq2k** [comment]
Is there something I'm missing? Navigating a book is pretty hard without splitting. If I accidentally hit a button on my player and skip a track I have no idea where I left off. How do you solve this problem when listening to books?
________________________________

**Comment by studersi** [comment]
+1 for automatically splitting

**Comment by mediacowboy** [comment]
I too will add support for automatic splitting

**Comment by openaudible** [comment]
Do you want auto-split and also the ability to auto-delete the source single mp3/m4b?

**Reply by maphew** [reply]
Yes, provided the original .aax is still present (so a single mp3/m4b can be recreated if needed).

**Comment by oboe45** [comment]
No, neither add anything to my workflow - usually I do not split  books and
I definitely do not want anything auto deleted!
